app/src/main/java/com/infomaniak/mail/ui/login/LoginActivity.kt

```diff
@@ -215,8 +227,14 @@ class LoginActivity : AppCompatActivity() {
         (getChildAt(0) as? RecyclerView)?.overScrollMode = View.OVER_SCROLL_NEVER
     }
 
+    private suspend fun launchNoMailboxActivity() = withContext(Dispatchers.Main) {
+        Intent(this@LoginActivity, NoMailboxActivity::class.java).apply { startActivity(this) }
+        binding.connectButton.hideProgress(R.string.buttonLogin)
+        binding.signInButton.isEnabled = true
```

Comment:
It it used to hide the progress animation of the login button (moving dots), and disable the other button during the time before the new activity starts to avoid navigating to another screen in this launch time.